lesson10/public/app.js

```diff
@@ -0,0 +1,51 @@
+$('#submit').click(()=>{
+  const name = $('#name').val();
+  const comment = $('#comment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/nameCom',
+    data:JSON.stringify ({name, comment}),
+    success:(data)=>{
+      console.log('success:', data);
+    },
+    contentType: 'application/json'
+   })
+})
+
+/*chat function*/
+const chat =()=>{
+  $.get('/ncom.txt', (data)=>{
+    $('#messages').html(data);
+  })
+}
+window.setInterval(chat,1000);
+
+/* turn on camera*/
+const video = $('#video')[0];
+navigator.mediaDevices.getUserMedia({ video:true, audio: false})
+  .then((mediaStream)=>{
+    video.srcObject = mediaStream;
+    video.play();
+  }).catch((err)=>{
+    console.log("error: ",err);
+  })
+ 
+/*take a picture*/
+$('#picture').click(()=>{
+  const name = $('#name').val();
```

Comment:
You don't HAVE to change this one, but normally the same as functions, if you are only gonna use a value once then don't use a variable just use it. If you are only going to use a function once then there's no point of building a function just use it